### DISCUSSION TITLE
cache:warmup warnig after updating from symfony 5.2.3 to 5.3.13

### DISCUSSION THREAD

**Discussion by menhir2015** [symfony/symfony#44869] [Q&A] [answered]
After updating from  symfony 5.2.3 to 5.3.13 and fixed various deprecations, if i do cache:warmup  from the cli, i got this issue

`15:03:13 WARNING   [app] Failed to generate ConfigBuilder for extension Symfony\Bundle\FrameworkBundle\DependencyInjection\FrameworkExtension. ["exception" => Symfony\Component\DependencyInjection\Exception\EnvNotFoundException^ { …},"extensionClass" => "Symfony\Bundle\FrameworkBundle\DependencyInjection\FrameworkExtension"]`
and others WARNINGS similar to this one

this is happening in both dev and prod env
according to this https://github.com/symfony/symfony/issues/41128
this should be already fixed 
i also deleted the vendor directory and installed new packages ,
any suggestions?
by the way i m using the new runtime component and changed the App/Kernel file and the public/index.php file according to the new component

**Comment by captsulu** [comment]
I am experiencing the same issues I am running Symfony 5.3

**Comment by lyrixx** [comment, marked as answer]
Hello, Could you create a simple reproducer and open a new issue? Thanks

**Reply by menhir2015** [reply]
Ok i found the problem, in my service.yaml i had this line
`parameters:
    app.keycloak_public_key: '%env(file:KEYCLOAK_PUBLIC_KEY)%'`
in my env file i had this
`KEYCLOAK_PUBLIC_KEY=../config/jwt/keycloak_public.pem`
has those lines are not used anymore i could easly remove them, 
the problem disappear if i use the correct path
`KEYCLOAK_PUBLIC_KEY=./config/jwt/keycloak_public.pem`
by the way this code was initially used and was working correct ( as git history is saying to me)

**Reply by menhir2015** [reply]
thank you for asking me to reproduce the error, it was the way to find the mistake
